spec/test-data/translations/ios_extract_keys_from_strings_files/WordPress-Resources/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,28 @@
+/* This file is supposed to be the file we send to GlotPress, typically after:
+    - generating the initial keys using `ios_generate_strings_file_from_code`
+    - merging the keys from other manually-maintained `.strings` files (e.g. `InfoPlist.strings` files from both WordPress + Jetpack)
+      using `ios_merge_strings_files` — using `wordpress.infoplist.` and `jetpack.infoplist.` key prefixes during the merge in this example.
+*/
+
+"key1" = "value1";
+"key2" = "value2";
+"key3" = "value3";
+
+/* MARK: - InfoPlist.strings */
+
+"wordpress.infoplist.NSLocationUsageDescription" = "WordPress would like to add your location to posts on sites where you have enabled geotagging.";
+"wordpress.infoplist.NSCameraUsageDescription" = "To take photos or videos to use in your posts.";
+
+/* MARK: - InfoPlist.strings */
+
+"jetpack.infoplist.NSLocationUsageDescription" = "Jetpack would like to add your location to posts on sites where you have enabled geotagging.";
+"jetpack.infoplist.NSCameraUsageDescription" = "To take photos or videos to use in your posts.";
+
+/* MARK: - Sites.strings */
+
+"BOl9KQ" = "There are ${count} options matching ‘${site}’.";
+"ILcGmf" = "Site";
+"cyajMn" = "Site";
+"gpCwrM" = "Select Site";
+"s4dJhx" = "Just to confirm, you wanted ‘${site}’?";
```

Comment:
💡  Maybe it could be worth to extend the Unit Tests to also use a prefix for those keys (like `ios-widget.`) and validate that they are also extracted in the `Sites.strings` files unprefixed (just like for the case of the 2 `InfoPlist.strings` files above)?

 - On one end, if the current test already proves it works for the case of the 2 `InfoPlist.strings` files, no reason it wouldn't work for `Sites.strings`, and we could thus decide to even remove those keys in that fixture.
 - On the other end, since I kept those keys in that fixture (mostly because I did a simple copy/paste of the similar-but-unprefixed fixture from `Resources/` to create that file, and did not think about removing those afterwards), why not take the occasion to use them and make our test cover more examples… 🤔

Reply:
Ok was quite quick to adjust the test to include them, so I already did it in 50d0a597331399611e7806af705d69e187e2d997 after all.